autopush/tests/test_endpoint.py

```diff
@@ -0,0 +1,486 @@
+import json
+import functools
+
+from cyclone.web import Application
+import twisted.internet.base
+from twisted.internet.defer import Deferred
+from txstatsd.metrics.metrics import Metrics
+from cryptography.fernet import Fernet, InvalidToken
+import requests
+from boto.dynamodb2.exceptions import (
+    ProvisionedThroughputExceededException,
+)
+
```

Comment:
No space here. Also, imports should be alphabetical in order, and imports/from's should be grouped.